pages/common/dolt-merge.md

```diff
@@ -0,0 +1,24 @@
+# dolt merge
+
+> Join two or more development histories together.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch:
+
+`dolt merge {{branch_name}}`
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch without updating the commit history:
+
+`dolt merge --squash {{branch_name}}`
+
+- Create a merge commit even when the merge resolves as a fast-forward:
+
+`dolt merge --no-ff {{branch_name}}`
+
+- Use the given {{message}} as the commit message:
+
+`dolt merge --no-ff -m {{message}} {{branch_name}}`
```

Comment:
```suggestion
`dolt merge --no-ff -m "{{message}}" {{branch_name}}`
```

Reply:
solved in 3ed2e40c70ce3daaf573c04d92a243a4fe566c7a